dulwich/porcelain.py

```diff
@@ -1045,11 +1048,12 @@ def branch_list(repo):
         return r.refs.keys(base=b"refs/heads/")
 
 
-def fetch(repo, remote_location, outstream=sys.stdout,
+def fetch(repo, remote_name, remote_location, outstream=sys.stdout,
```

Comment:
This is an API change, could you perhaps make remote_name an optional argument that defaults to None (in which case the original behaviour is preserved)?

Reply:
No problem; my upcoming change will actually set it to "b'origin'", since that avoids an unnecessary conditional in the code, if that's okay?